docs/example-react-intl.mdx

```diff
@@ -55,7 +55,7 @@ You have two options:
        import 'intl/locale-data/jsonp/pt'
 
        export const setupTests = () => {
-         // https://formatjs.io/guides/runtime-environments/#server
+         // https://formatjs.io/docs/guides/runtime-requirements
```

Comment:
Should this point to the [Node.js section](https://formatjs.io/docs/guides/runtime-requirements/#nodejs) considering the previous section targeted server-side rendering?

Reply:
Sounds great, i'll change it to point Node.js section.